### DISCUSSION TITLE
How to parameterize a closed mesh?

### DISCUSSION THREAD

**Discussion by breakerrorchen** [CGAL/cgal#6748] [Q&A] [answered]
I have another closed surface and no border. Is there any way to parameterize it?

**Comment by MaelRL** [comment, marked as answer]
You can either use a parameterization method which is designed for topological balls ([Orbifold_Tutte_parameterizer_3](https://doc.cgal.org/latest/Surface_mesh_parameterization/classCGAL_1_1Surface__mesh__parameterization_1_1Orbifold__Tutte__parameterizer__3.html)) or create artificial, combinatorial boundaries on your mesh with the class [`CGAL::Seam_mesh`](https://doc.cgal.org/latest/BGL/classCGAL_1_1Seam__mesh.html) such that it becomes a topological disk, see [this example](https://doc.cgal.org/latest/Surface_mesh_parameterization/Surface_mesh_parameterization_2seam_Polyhedron_3_8cpp-example.html).
